### PR TITLE
Edited the text

### PR DESCRIPTION
**Pull request by WheatMan999**
Hey guys, I dared to make some minor changes in text: edited misspelling, redundant colons, a/the confusions, etc. Will be cool, if it helps you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,14 +61,14 @@ The course includes links to different learning materials - video courses, tutor
...
-We would like to know the progress of students along the way. We kindly ask you to post about you progress (finished tasks) in students chat.
+We would like to know the progress of students along the way. We kindly ask you to post about your progress (finished tasks) in students chat.
 
  ### Chats
 
 For your comfort we have created several telegram-chats:
 
 1. [FE Students | Kottans](https://t.me/joinchat/DmX0JBHVkEhV1us2HdMmpA) - chat for students, where you can discuss anything related to the current course
-1. [FE Questionarium | Kottans](https://t.me/joinchat/DmX0JAl-mh5W0jrWli8Ycw) - if you need a help of one of the mentors, or maybe from all of them, you can ask there
+1. [FE Questionarium | Kottans](https://t.me/joinchat/DmX0JAl-mh5W0jrWli8Ycw) - if you need the help of one of the mentors, or maybe from all of them, you can ask there
```
Comment (miroslav-kolomiets) [inline README.md:71]:
```suggestion
1. [FE Questionarium | Kottans](https://t.me/joinchat/DmX0JAl-mh5W0jrWli8Ycw) - if you need help of one of the mentors, or maybe from all of them, you can ask there
```

Comment (miroslav-kolomiets) [reply README.md:71]: I'm not 100 percent sure, but it's probably better this way.

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Thanks, nice job! But some moments needs discussion.

File: README.md (hunk 1 above)
Comment (WheatMan999) [reply README.md:71]: Yeah, tricky one. Agree, it's better without a/the. Should I update it in my local copy and make a request again?

Comment (zonzujiro) [reply README.md:71]: @Andrew-Omelchenko you can, if you want. And then make a force push into this PR.

Or owner of repo can just press "Commit suggestion".

**Review by zonzujiro — APPROVED** [review]

**Review by miroslav-kolomiets — APPROVED** [review]
Approve:)

**Pull request merged**